mne/utils/docs.py

```diff
@@ -45,10 +50,23 @@
     interpreted as channel indices. None (default) will pick all channels.
 """
 
+# Rank
+docdict['rank'] = """
+rank : None | dict | 'info' | 'full'
+        This controls the rank computation that can be read from the
+        measurement info or estimated from the data. See ``Notes``
+        of :func:`mne.compute_rank` for details."""
+docdict['rank_None'] = docdict['rank'] + 'The default is None.'
+docdict['rank_info'] = docdict['rank'] + 'The default is "info".'
```

Comment:
should we fight to have the same default everywhere?

Reply:
Probably, but that's a fight for another day :)